packages/cli/src/controllers/folder.controller.ts

```diff
@@ -48,4 +48,23 @@ export class ProjectController {
 			throw new InternalServerError();
 		}
 	}
+
+	@Patch('/:folderId')
+	@ProjectScope('folder:update')
+	async updateFolder(
+		req: AuthenticatedRequest<{ projectId: string; folderId: string }>,
```

Comment:
This is of a question than review comment, we have the decorator for `@Param` that could be used to read `folderId` and `projectId` params, but AuthenticatedRequest also lets you set the RouteParams like you've done here. Do we have preference over which way to use?
It looks like the decorator just reads the given key from `req.params[key]` and I can't spot any additional logic on it, so this probably isn't important?

Reply:
Good point. The decorators are kind of new, so might have been more cleaning to use the @param in this case. Will address in a follow up PR.